Sage/carbon
# WIP docs(select): add aria label prop documentation (FE-5662)

### Proposed behaviour

add `aria-label` and `aria-labelledby` as props in `PropTypes` to all select components

### Current behaviour

in select components `aria-label` and `aria-labelledby` attributes could be passed down to the input, but props for it are not documented 

### Checklist



~~- [ ] Screenshots are included in the PR if useful~~
~~- [ ] All themes are supported if required~~
~~- [ ] Storybook added or updated if required~~
~~- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required~~

#### QA


### Additional context

closes #5783

### Testing instructions

props for those ARIA attributes should be displayed in all select stories

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
